python/triton/backends/compiler.py

```diff
@@ -210,6 +210,9 @@ def get_property_key(val, align):
             return "1"
         return "N"
 
+    def __repr__(self):
+        return f"AttrsDescriptor.from_dict({self.to_dict()})"
```

Comment:
Don't think it makes a difference currently, but this should probably format the dict's`repr`
```suggestion
        return f"AttrsDescriptor.from_dict({self.to_dict()!r})"
```

Reply:
Makes sense. Done.